### DISCUSSION TITLE
tactical rmm ERROR: Only x86_64 arch is supported, not aarch64

### DISCUSSION THREAD

**Discussion by ITCMD** [amidaware/tacticalrmm#1516] [General]
**Server Info (please complete the following information):**
 - OS: Ubuntu 20.04.6 LTS aarch64
 - Browser: None (SSH)
 - Not yet installed

**Installation Method:**
  - [x] Standard
  - [ ] Docker

Im following the traditional install setup, and on running the install script:

```
wget https://raw.githubusercontent.com/amidaware/tacticalrmm/master/install.sh
chmod +x install.sh
./install.sh
```
It responds with: `tactical rmm ERROR: Only x86_64 arch is supported, not aarch64`

I would assume aarch64 is 64x arch, am I incorrect? Is there any way to get it running on this server as it's the only version of Ubuntu 20.04 they support.

**Comment by dinger1986** [comment]
aarch64 is arm.

Where are you trying to install? Oracle free tier?

**Reply by ITCMD** [reply]
Yes, that is correct, it's on the Ampere A1 Compute Instances. Very powerful, so I figured it'd make sense. I'm guessing that means no support? I haven't run into any other software that hasn't run so far on it.

**Reply by dinger1986** [reply]
Arm isn't supported hence the check. You could edit the install script to use arm architecture but would need to compile nats-api for the correct arm cpu version. I'd suggest using Postgres for mesh db as well.
